clang/test/lab1/veselov_mikhail/test.cpp

```diff
@@ -0,0 +1,223 @@
+// RUN: split-file %s %t
+
+// RUN: %clang_cc1 -load %llvmshlibdir/VeselRenamePlugin%pluginext\
+// RUN: -add-plugin rename\
+// RUN: -plugin-arg-rename type=var\
+// RUN: -plugin-arg-rename cur-name=a\
+// RUN: -plugin-arg-rename new-name=new_var %t/rename_var.cpp
+// RUN: FileCheck %s < %t/rename_var.cpp --check-prefix=VAR
+
+//--- rename_var.cpp
+
+// VAR: int func() {
+// VAR-NEXT: int new_var = 2, b = 2;
+// VAR-NEXT: new_var = b + new_var;
+// VAR-NEXT: new_var++;
+// VAR-NEXT:  return new_var;
+// VAR-NEXT: }
+
+int func() {
+  int a = 2, b = 2;
+  a = b + a;
+  a++;
+  return a;
+}
+
+// RUN: %clang_cc1 -load %llvmshlibdir/VeselRenamePlugin%pluginext\
+// RUN: -add-plugin rename\
+// RUN: -plugin-arg-rename type=var\
+// RUN: -plugin-arg-rename cur-name=c\
+// RUN: -plugin-arg-rename new-name=new_var %t/rename_non_existent_var.cpp
+// RUN: FileCheck %s < %t/rename_non_existent_var.cpp --check-prefix=NON_EXIST_VAR
+
+//--- rename_non_existent_var.cpp
+
+// NON_EXIST_VAR: int func() {
+// NON_EXIST_VAR-NEXT: int a = 2;
+// NON_EXIST_VAR-NEXT: int b = 3;
```

Comment:
```suggestion
// NON_EXIST_VAR-NEXT: int bbbb = 3;
```
Then let's look if this test fails or not.

Reply:
Do you mean that you need to rename all variables "b" to "bbbb" in the code we handle with our plugin?

Reply:
I meant to validate on your own if those tests are executed at all or not.

Reply:
I changed it, it crashes for me, do I need to upload it to the pool request?

Reply:
Hmm, OK then. No, do not change the test.